CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewPane.java

```diff
@@ -180,156 +193,155 @@ public void onChanged(final ListChangeListener.Change event) {
+     *
+     * @param notesViewEntries A list of NotesViewEntry objects to add to
+     * notesViewEntries.
      */
-    protected synchronized void setNotes(final List<NotesViewEntry> notesViewEntries) {
+    protected void setNotes(final List<NotesViewEntry> notesViewEntries) {
         Platform.runLater(() -> {
-            this.notesViewEntries.clear();
-            
-            notesViewEntries.forEach(entry -> {
-                this.notesViewEntries.add(entry);
-            });
-            
-            updateNotes();
+            synchronized (LOCK) {
+                this.notesViewEntries.clear();
+
+                notesViewEntries.forEach(note -> {
+                    addNote(note);
+                });
+            }
+
+            updateNotesUI();
         });
     }
-    
+
     /**
      * Sets selectedFilters.
-     * 
+     *
      * @param selectFilters A list of String objects to add to selectedFilters.
      */
-    protected synchronized void setFilters(final List<String> selectedFilters) {
+    protected void setFilters(final List<String> selectedFilters) {
         Platform.runLater(() -> {
             this.selectedFilters.clear();
-            
+
             selectedFilters.forEach(filter -> {
                 this.selectedFilters.add(filter);
             });
-            
+
             updateFilters();
         });
     }
-    
+
     /**
      * Updates the UI of the notes currently being displayed in the Notes View.
      */
-    protected synchronized void updateNotes() {
+    protected synchronized void updateNotesUI() {
         Platform.runLater(() -> {
             notesListVBox.getChildren().removeAll(notesListVBox.getChildren());
-            
+
             final List<NotesViewEntry> notesToRender = new ArrayList<>();
-            
-            notesViewEntries.forEach(entry -> {
-                // Add user note to render list if "User Note" filter is selected.
-                if (selectedFilters.contains("User Notes") && entry.isUserCreated()) {
-                    notesToRender.add(entry);
-                }
-                // Add auto note to render list if "Auto Note" filter is selected.
-                if (selectedFilters.contains("Auto Notes") && !entry.isUserCreated()) {
-                    notesToRender.add(entry);
-                }
-            });
-            
+
+            synchronized (LOCK) {
+                notesViewEntries.forEach(entry -> {
+                    // Add user note to render list if "User Note" filter is selected.
+                    if (selectedFilters.contains(USER_NOTES_FILTER) && entry.isUserCreated()) {
+                        notesToRender.add(entry);
+                    }
+                    // Add auto note to render list if "Auto Note" filter is selected.
+                    if (selectedFilters.contains(AUTO_NOTES_FILTER) && !entry.isUserCreated()) {
+                        notesToRender.add(entry);
+                    }
```

Comment:
can probably merge the two if statements (with `||`) given they have the same code body